subsys/zigbee/osif/zb_nrf_logger.c

```diff
@@ -9,55 +9,174 @@
 #if defined ZB_NRF_TRACE
 
 #include <logging/log.h>
+#include <sys/ring_buffer.h>
```

Comment:
Do we want to mix two, mutually exclusive logging backends in a single source file?